src/main/java/org/folio/services/lines/PoLinesService.java

```diff
@@ -221,6 +223,12 @@ public Future<List<PoLine>> getPoLinesByHoldingId(String holdingId, Conn conn) {
     return getPoLinesByField(criterion, conn);
   }
 
+  public Future<List<PoLine>> getPoLinesByCqlQuery(String query, Conn conn) throws FieldException {
```

Comment:
Added to as a potential replacement of `getPoLinesByHoldingId()`, SQL with `location.holdingId` does not return any rows